Makefile

```diff
@@ -64,3 +64,11 @@ upload:
 list:
 	@grep '^[^\.#[:space:]].*:' Makefile | \
 		cut -d':' -f1
+
+new_connector:  # $ make new_connector type=Magento
```

Comment:
to avoid issue if `type` is empty you can add the following Makefile test:

```
ifeq ($(type),)
        @echo -e "Missing type option:\n\tmake $@ type=MyConnecter"
        exit 1
endif
```

otherwise that seems nice :+1: 

nota: we should decide if we use `$[type}` or `$(type)` in our bash variable :D previously William use one approach and I use the other ^^ we never take a decision about this point... :P

Reply:
should I go for `{VAR}` ? is it the one you are using ? 

I tried to copy and paste the parameter checking code above but I must have make a mistake, it gave me : 

```
make new_connector type=Denodo
ifeq (Denodo,)
/bin/bash: -c: line 0: syntax error near unexpected token `Denodo,'
/bin/bash: -c: line 0: `ifeq (Denodo,)'
```

Reply:
I prefer to use the `${VAR}` but I will use what you want :) we just need to use the same

about the `ifeq`, it's not a bash part but a Makefile part ^^ this is why
please refer to the following example: https://github.com/ToucanToco/latapult/blob/master/Makefile#L25